cluster/images/conformance/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
Do we need to copy in cluster/images/conformance? I'd like to minimize what we import.